messaging/src/main/java/org/axonframework/messaging/annotation/AggregateType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
```

Comment:
A very tiny thing, but it should be the current year, so 2022 instead of 2018, also for the other new files.